sim/cfg/nos3-eps-simulator.xml

```diff
@@ -24,9 +24,23 @@
                     <bus>
                         <battery-voltage>24.0</battery-voltage>
                         <battery-temperature>30.0</battery-temperature>
+                        <battery-watt-hours>10.00</battery-watt-hours>
                         <solar-array-voltage>32.0</solar-array-voltage>
+                        <solar-array-current>4.0</solar-array-current>
                         <solar-array-temperature>80.0</solar-array-temperature>
```

Comment:
Is temperature used somewhere?  If not, consider removing it.

Reply:
Temperature was there to begin with (and unused); I would imagine that leaving it could be useful if we want to change the power depending on varying temperatures in the future. Right now it serves no purpose, though.